github/TimelineEvent.py

```diff
@@ -86,16 +88,28 @@ def source(self) -> github.TimelineEventSource.TimelineEventSource | None:
 
     @property
     def body(self) -> str | None:
-        if self.event == "commented" and self._body is not NotSet:
+        if is_defined(self._body):
```

Comment:
I think we don't need to check if it's defined, `NotSet` itself is already `Attribute[None]`, you can just return its `.value`.